docs/source/reference/api.rst

```diff
@@ -3,6 +3,66 @@ LiberTEM Context API
 
 See :ref:`api documentation` for introduction and complete examples.
 
+Context
+~~~~~~~
+
 .. automodule:: libertem.api
    :members:
    :special-members: __init__
+
+Job API
+~~~~~~~
+
+.. automodule:: libertem.job.base
+   :members:
+   :special-members: __init__
+
+MaskContainer and ApplyMaskjob
```

Comment:
Typo (was already there in old version): `ApplyMasksJob`.

There should be a note that this part (`Job` and `Analysis`) of the API is actually internal, and subject to change. That was the idea of having the `Context`, to allow changes of the implementation without impacting users.

Reply:
Ok, the Job is now not included anymore. The MaskContainer is already in use in the SparseCorrelationUDF and it seems like a generally useful component. Do you think we'll have to change much and warn the user about it or can we publish it as part of the API?

Reply:
After consideration, I've moved it back to the Internal part since it is not used in the API. Is there a reason to have the ApplyMasksJob documented at all?

Reply:
I would need to have a closer look at `MaskContainer` again, but for now I would prefer to have it internal. Maybe we should revisit as a general tool to use, once the parameters are properly scattered and shared in shm?

I agree that having `Job` impls documented is not really useful, also because the interface is very simple.

Reply:
Ok, then I'll leave it as in the last commit where I made `MaskContainer` internal! From `Job` only the abstract base class remains in the API documentation because it is the return type of some `Context` methods.